move_tab.py

```diff
@@ -12,51 +12,27 @@
 
 http://github.com/FMCorz/MoveTab
 """
+import sublime_plugin
 
-import sublime, sublime_plugin
 
 class MoveTabCommand(sublime_plugin.WindowCommand):
 
-	def run(self, position):
-		position = str(position)
-		view = self.window.active_view()
-		(group, index) = self.window.get_view_index(view)
-		if index < 0:
-			return
-		count = len(self.window.views_in_group(group))
-
-		direction = None
-		if position.startswith('-') or position.startswith('+'):
-			direction = position[0]
-			steps = int(position[1:])
-			if direction == '-':
-				position = index - steps
-			else:
-				position = index + steps
-
-		position = int(position)
-		if position < 0:
-			position = count - 1
-		elif position > count - 1:
-			if direction: position = 0
-			else: position = count - 1
-
-		# Avoid flashing tab when moving to same index
-		if position == index:
-			return
-
-		self.window.set_view_index(view, group, position)
-		self.window.focus_view(view)
-
-	def is_enabled(self):
-		view = self.window.active_view()
-		if view == None:
-			return False
-		(group, index) = self.window.get_view_index(view)
-		return len(self.window.views_in_group(group)) > 1
```

Comment:
The check for whether a group contains more than one view has been removed. Was this intentional?